packages/fast-usdc-contract/test/contract-setup.ts

```diff
@@ -432,43 +433,45 @@ export const makeCustomer = (
       await eventLoopIteration();
       return tx;
     },
+
     checkSent: (
       t: ExecutionContext<FucContext>,
       { bank = [] as any[], local = [] as any[] } = {},
-      forward?: unknown,
+      { forward = false, route = 'pfm' as 'pfm' | 'cctp' } = {},
     ) => {
       const next = sent.shift();
       if (!next) throw t.fail('nothing sent');
       const { evidence } = next;
+      const { EUD } = decodeAddressHook(evidence.aux.recipientAddress).query;
```

Comment:
https://github.com/Agoric/agoric-sdk/blob/c7f5da38b817741ce55d815289981834917b1d4a/packages/fast-usdc/src/type-guards.js#L104-L108

```suggestion
      const { EUD } = decodeAddressHook(evidence.aux.recipientAddress).query;
      mustMatch(decoded, AddressHookShape);
```

Then you shouldn't need the cast below